crates/infra/cli/src/commands/run/mod.rs

```diff
@@ -19,26 +19,42 @@ enum RunCommand {
      * User-facing:
      *
      */
-    /// Run the public 'slang_solidity' crate shipped to Cargo users.
+    /// Runs the public 'slang_solidity' crate shipped to Cargo users.
     #[clap(name = "slang_solidity")]
     SlangSolidity,
-
-    /// Run smoke tests of the Solidity parser against source files from the Sanctuary repositories.
-    #[clap(name = "solidity_testing_smoke")]
-    SolidityTestingSmoke,
+    /// Runs the Solidity parser against source files from the Sanctuary repositories.
+    #[clap(name = "solidity_testing_sanctuary")]
+    SolidityTestingSanctuary,
 
     /*
      *
      * Hidden (for automation only):
      *
      */
+    /// Runs codegen for the Rust parser crate.
     #[clap(name = "solidity_cargo_build", hide = true)]
     SolidityCargoBuild,
-
+    /// Runs codegen for the NAPI parser crate.
```

Comment:
nit: The wording seems to imply that NAPI may somehow correspond equally to Rust but it's the "same" Rust parser crate, just with aux bindings/glue to NAPI. What do you think about clarifying this a little bit?
```suggestion
    /// Runs codegen for the NAPI-exposed parser crate.
```

Reply:
Sure.